pkg/gateway/model/deployment_route.go

```diff
@@ -32,3 +33,19 @@ func (r RouteTypeConfig) TargetPath() string {
 	}
 	return ""
 }
+
+func (r RouteTypeConfig) AssetPathMapping() map[string]string {
+	m := r["asset_path_mapping"].(map[string]interface{})
```

Comment:
Can you add some explanation how we construct `asset_path_mapping`?

Reply:
You can see the `deployment_route_test.go` file for reference on how `static` deployment route works. Copied below for reference:
```go
{
	url:       "https://example.com/login#form",
	matchPath: "/index-html",
	route: config.DeploymentRoute{
		Type: "static",
		Path: "/",
		TypeConfig: map[string]interface{}{
			"backend_url": "https://app.localhost/_asset/get",
			"asset_path_mapping": map[string]interface{}{
				"/index.html":              "index-html",
				"/assets/main.12345678.js": "main-js",
			},
			"asset_fallback_path": "/",
		},
	},
	forwardURL: "https://app.localhost/_asset/get/index-html",
}
```

`asset_fallback_path` corresponds to `fallback` config in `skygear.yaml`.
`asset_path_mapping` is a map with URL absolute path (based on deployment route path) as key, and asset name as value.

Controller would fill in the get asset endpoint of app asset gear as `backend_url`, and upload individual static assets to it.